etc/dbus-serialbattery/dbushelper.py

```diff
@@ -395,8 +396,8 @@ def publish_dbus(self):
             "/Info/MaxDischargeCurrent"
         ] = self.battery.control_discharge_current
 
-        # Voltage control
-        self._dbusservice["/Info/MaxChargeVoltage"] = self.battery.control_voltage
+        self._dbusservice["/Info/BatteryLowVoltage"] = self.battery.control_discharge_voltage
```

Comment:
Is this working?

> `/Info/BatteryLowVoltage     <- Note that Low Voltage is ignored by the system `

See https://github.com/victronenergy/venus/wiki/dbus#battery